language-design.md

```diff
@@ -16,8 +16,22 @@ are:
 For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
+"We have to do this a lot when we write real computer programs: a thought that seems like a primitive in our minds turns out not to be a primitive in a programming language, and in each new program we must define it once more." [Steele, 1998]
 
+I chose this quote because I thought it was epecially true and interesting. It highlights the differences in various programming languages. Perhaps if you take some primitives to be more important than others then you will perfer languages with those defined primitives. -Daniel
 
+I've certainly found this to be the case in my experience programming. I'd also add that what does or doesn't turn out to be a primitive, as well as what you might expect to be a primitive, depends in some capacity on what you're used to. For a while after taking CS5, for example, I'd tend to want languages like Java to have as primitives bits of functionality that are primitives in Python. One specific example that comes to mind is that in Python, the built-in max (and min) function can be called
+on lists and other collections, whereas Java's Math package only has max and min methods that take two numbers as arguments. -Aaron
+
+"In a race, a small language with warts will beat a well designed language because users will not wait for the right thing; they will use the language that is quick and cheap, and put up with the warts." [Steele, 1998]
+
+I chose this quote for two reasons. First, I think it's true but second it is a frustrating concept to me as I am somewhat of a perfectionist. I also do not think that choosing a poorly designed but easy to adopt language is better in the long run. In other words, I think it is the wrong choice. But people seem to choose it all the same. -Daniel
+
```

Comment:
I think a major argument Steele tries to make is that quick deliverables are crucial for the success of a language. I agree that it is frustrating as a developer, but imagine from the user perspective. Just how easily is it to find these warts? For me personally, I think it's extremely easy to recognize the limitations of a particular language but not the specific "warts" of the language.